File: factorial.py
print("Factorial de un numero")
n = int(input("Ingrese un numero "))
factorial = 1

if n == 1:
	print("El factorial de ",n," es ",factorial)

else:
	for i in range(1,n+1):
		factorial = factorial*i

print("El factorial de ",n," es ",factorial)

File: factorial.cpp
#include <iostream>
using namespace std;
int main(){
	int n,factorial=1;
	cout<<"Ingrese el numero para calcular su factorial"<<endl;
	cin>>n;
	for(int i=1;i<=n;i++){
		factorial *= i;
	}
	
	cout<<"El factorial de tu numero es: "<<factorial<<endl;
	
	
	
	
	return 0;
}
